packages/components/src/VirtualizedList/CellIconText/CellIconText.component.js

```diff
@@ -30,13 +55,31 @@ class CellIconText extends React.Component {
 	}
 
 	render() {
+		const { columnData = {} } = this.props;
 		const icon = getCellIcon(this.props);
 		const label = getCellLabel(this.props);
+		const labelTooltip = getCellLabelTooltip(this.props);
+		const iconTooltip = getCellIconTooltip(this.props);
 
 		return (
 			<div className={css('tc-icon-text')}>
-				{icon && <Icon name={icon} />}
-				<span className={theme.label}>{label}</span>
+				{icon &&
+					(iconTooltip ? (
+						<TooltipTrigger
+							label={iconTooltip}
+							tooltipPlacement={columnData.tooltipPlacement || DEFAULT_TOOLTIP_PLACEMENT}
+						>
+							<span><Icon name={icon} /></span>
+						</TooltipTrigger>
+					) : (
+						<Icon name={icon} />
+					))}
+				<TooltipTrigger
+					label={labelTooltip || label}
```

Comment:
I will always have a tooltip here ?

Reply:
One is a custom which user can define (labelTooltip, the other one is a default which is a cell text)